src/components/Chip.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import css from '@styled-system/css';
+import { Box as BoxPrimitive, BoxProps as BoxPrimitiveProps } from 'mdlz-prmtz';
+import { variant, Prop } from '@modulz/radix-system';
+import { transparentize } from 'polished';
+import {theme} from '../theme';
+
+type ChipProps = BoxPrimitiveProps & {
+  variant?: Prop<'gray' | 'primary' | 'blue' | 'green' | 'purple' | 'orange'>;
+  size?: Prop<0 | 1 | 2>;
+  truncate?: Prop<boolean>;
+};
+
+export const Chip = React.forwardRef<HTMLDivElement, ChipProps>((props, ref) => (
+  <BoxPrimitive
+    {...props}
+    ref={ref}
+    css={[
+      css({
+        fontSize: 'inherit',
+        fontWeight: 600,
+        margin: '4px',
+        alignSelf: 'flex-start',
+        display: 'inline-block',
+        minWidth: '20px',
+        textAlign: 'center',
+        borderRadius: '12px',
+      }),
+      variant({
+        variant: {
+          gray: {
+            color: 'grays.7',
+            backgroundColor: 'grays.2',
+          },
+          primary: {
+            color: 'primary',
+            backgroundColor: transparentize(0.85, theme.colors.primary),
+          },
+          blue: {
+            color: 'blue',
+            backgroundColor: 'blues.2',
+          },
+          green: {
+            color: 'green',
+            backgroundColor: 'greens.2',
+          },
+          purple: {
+            color: 'purple',
+            backgroundColor: 'purples.2',
+          },
+          orange: {
+            color: 'orange',
+            backgroundColor: 'oranges.2',
+          }
+        },
+      }),
+      variant({
+        size: {
+          0: {
+            fontSize: 0,
+            lineHeight: '20px',
+            letterSpacing: '.032em',
+            paddingX: '4px',
```

Comment:
PaddingX seems too small regarding the design and the current Traefik dashboard implemention. In Traefik it's 6px. Maybe we should either set it to 8px based on ou styled-system theme having `paddingX: 1` or set it to 10px.

Reply:
Sure, I adjusted the component to use mostly Theme values, it should be more consistent now.